expression/builtin_string.go

```diff
@@ -416,8 +416,12 @@ func (b *builtinConcatWSSig) evalString(row chunk.Row) (string, bool, error) {
 		strs = append(strs, val)
 	}
 
-	// TODO: check whether the length of result is larger than Flen
-	return strings.Join(strs, sep), false, nil
+	// check whether the length of result is larger than Flen
+	str := strings.Join(strs, sep)
+	if len(str) > b.tp.Flen {
```

Comment:
Why do we have to check this? It seems dangerous, for example, if the length is `unspecified` which is `-1`, it always returns an empty string.

Reply:
how about `if b.tp.Flen != unspecified && len(str) > b.tp.Flen` ? or I should not check there? L419 had a todo tag `, so I do this.

Reply:
Oh, I don't notice the TODO tag before.
I  think `b.tp.Flen != unspecified && len(str) > b.tp.Flen` is OK.

Reply:
OK I will fix it.